Source/WebCore/css/CSSProperties.json

```
@@ -3832,6 +3832,16 @@
                 "url": "https://www.w3.org/TR/css-overflow-3/#propdef-overflow"
             }
         },
+        "overflow-anchor": {
```

```suggestion
        "overflow-anchor": {
            "codegen-properties": {
                "settings-flag": "cssScrollAnchoringEnabled"
            },
```

This is how you link the property to the setting.

---

Ah I see, thanks.